src/main/java/com/shop/ShopOperations.java

```diff
@@ -0,0 +1,11 @@
+package com.shop;
+
+import com.shop.dto.Catalog;
```

Comment:
Let's have a following structure in the project:
1. src/main/java/com/shop <- in this root folder we have a Main class (where we have public static void main method)
2. .../shop/service <- here we are gonna store all service classes. For example CatalogService, CategoryService, ProductService. Try to think how they should be implemented (use interfaces for that because in future service could have more than one implementation)
3. .../shop/repository <- same as for services. It is very common to have a different implementations related on the source of storing
4.  .../shop/model <- in this case it can be DTO's 
5.  .../shop/util <- place for your util classes. Try to think, do you really need them for now or this logic could be done by appropriate service